custom/js/modules/network.js

```
@@ -2,133 +2,1279 @@ export default class NetworkModule {
+	setupDiagnostics() {
+		document.getElementById('ping-btn')?.addEventListener('click', () => this.runDiagnostic('ping'));
+		document.getElementById('traceroute-btn')?.addEventListener('click', () => this.runDiagnostic('traceroute'));
+		document.getElementById('wol-btn')?.addEventListener('click', () => this.runWoL());
 	}
 
 	cleanup() {
 		if (this.subTabs) {
 			this.subTabs.cleanup();
 			this.subTabs = null;
 		}
+		this.cleanups.filter(Boolean).forEach(fn => fn());
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)